storage/sql_migrations/002_discoveryservice.sql

```diff
@@ -1,25 +1,20 @@
 -- +goose ENVSUB ON
 -- +goose Up
--- discovery contains the known discovery services and the associated tags.
+-- discovery contains the known discovery services and the associated timestamp.
 create table discovery_service
 (
     -- id is the unique identifier for the service. It comes from the service definition.
-    id varchar(200)         not null primary key,
-    -- tag is the latest tag pointing to the last presentation registered on the service.
-    last_tag   varchar(40)  null,
-    -- tag_prefix is used to prefix the tag of the presentations of the service.
-    -- It is only populated if the node is server for this service.
-    tag_prefix varchar(5)   null
+    id varchar(200)  not null primary key,
+    -- last_timestamp is the latest lamport_timestamp pointing to the last presentation registered on the service.
+    last_timestamp   integer  not null
```

Comment:
in discovery_presentation this is called `lamport_timestamp`. Call it `last_lamport_timestamp` here, or `timestamp` in the other table?